source/include/TypeAndDataManager.h

```diff
@@ -1304,6 +1305,21 @@ class StackVar
     }
 };
 
+struct StringRefCmp {
```

Comment:
The code appears to be a copy from the one in QueueRefNumManager in Synchronization.cpp, Is there a way we can share it?

Reply:
Missed it. Good catch. Thanks. Fixed in new PR.